lib/app/modules/uscore_v3.0.0/us_core_patient_sequence.rb

```diff
@@ -7,7 +7,7 @@ class USCore300PatientSequence < SequenceBase
 
       description 'Verify that Patient resources on the FHIR server follow the Argonaut Data Query Implementation Guide'
 
-      test_id_prefix 'Patient' # change me
+      test_id_prefix 'USCP'
```

Comment:
Does it matter if patient,practitioner, and procedure have the same prefix?

Reply:
Ok, that is a concrete example of the issue brought up by @Jammjammjamm.  I'll put this back in WIP and will address this.